### ISSUE TITLE
MPT: Review Spec

### ISSUE THREAD

**Issue by aguzmant103** [privacy-scaling-explorations/zkevm-specs#289]

**Comment by miha-stopar** [comment]
MPT docs are now cleaned, I think the PR can be reviewed now.

**Comment by aguzmant103** [comment]
@miha-stopar  to send the PR when possible

**Comment by miha-stopar** [comment]
Could be reviewed now:

- onExistingStorage proof docs were added
- [matching](https://github.com/privacy-scaling-explorations/zkevm-specs/blob/5cda455650b27d25dc9e3472c387f4b3e1942c3d/specs/mpt/scope.md) with the scoping document added

**Comment by aguzmant103** [comment]
Next steps: Ready to be reviewed by us first and then ask someone from Taiko.

**Comment by aguzmant103** [comment]
@Brechtpd we have this task on our backlog to review MPT Specs. By any chance are you interested in taking this one?

**Comment by Brechtpd** [comment]
Sure thing, can start working on this next week.

**Comment by miha-stopar** [comment]
> Sure thing, can start working on this next week.

@Brechtpd if it helps we can have a call before you begin, maybe just to go briefly over the content.

**Comment by aguzmant103** [comment]
Yes, thank you so much! @Brechtpd also helps with alignment on overall approach

**Comment by ChihChengLiang** [comment]
Will be updated once the witness generator work is finished.

**Comment by ed255** [comment]
The spec needs to be rewritten according to the refactor work done on the circuit implementation.  Miha will work on it after the witness generator work is finished.

**Comment by ChihChengLiang** [comment]
The spec needs to be updated to reflect the changes in the refactor.
